test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs

```diff
@@ -191,6 +190,40 @@ public void RestoreCommand_FromPackagesConfigFile()
             }
         }
 
+        [Fact]
+        public async void RestoreCommand_MissingNuspecFileInPackage()
```

Comment:
naming: 
Try to include the outcome in the name. 

For example:
RestoreCommand_MissingNuspecFileInPackage_FailsWithNUCode